src/entropy.jl

```diff
@@ -46,6 +46,9 @@ Currently implemented estimators are:
 - [`Kraskov`](@ref)
 - [`Zhu`](@ref)
 - [`ZhuSingh`](@ref)
+- [`Goria`](@ref)
+- [`Gao`](@ref)
```

Comment:
What's up with this diff? Are we adding one or several estimators in this PR?

Reply:
These estimators were missing from the list (forgot it in the last PRs), so I added them.